src/parser/jobs/ast/index.tsx

```diff
@@ -30,13 +30,20 @@ export default new Meta({
 	</>,
 	supportedPatches: {
 		from: '5.05',
-		to: '5.4',
+		to: '5.5',
```

Comment:
I'm pretty sure Earthly star is still bugging out a little at this point, I was halfway through a fix before I had to go on hiatus. But regardless I agree it's fine to bump since there really were no changes since like ~5.3

Reply:
good to know!
please let me know if you'd like help with that earthly star fix